tests/gui/models/supported-formats.spec.js

```diff
@@ -61,6 +62,17 @@ describe('Browser: SupportedFormats', function() {
 
     describe('.isSupportedImage()', function() {
 
+      _.forEach(imageStream.supportedFileTypes, (fileType) => {
```

Comment:
To match up with the style of the other unit tests, I think this should be explicitly spelled out, rather than just imported from `imageStream` ?  @jviotti to confirm :)

(and would that also prevent the "but that still won't save us from making that logical error in this setup" you refer to?)